commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQL.java

```diff
@@ -514,15 +514,25 @@ public static PalantirSqlException handleInterruptions(long startTime,
             throw PalantirSqlException.create(cause);
         }
         String msg = "We got an execution exception that was an interrupt, most likely from someone " + //$NON-NLS-1$
-                "incorrectly ignoring an interrupt. "; //$NON-NLS-1$
+                "incorrectly ignoring an interrupt. Elaped time: {} \nError message: {} "
```

Comment:
Always logging ``Elapsed time``, value will be ``N/A`` if it is not available. Also, inlined the ``msg``.